resources/helm/v1.1/istio/charts/kiali/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 description: Kiali is an open source project for service mesh observability, refer to https://www.kiali.io for details.
 name: kiali
-version: 1.9.1
```

Comment:
was this change intentional? it doesn't look like anything but the version number changed here

Reply:
This change was pulled in by the Makefile, let me check what is going on

Reply:
cleaning everything out and rerunning the Makefile fixes this, I'll push up a new version now